docs/UserGuide.md

```diff
@@ -766,7 +767,27 @@ Expected outcome:
      _________________________________________________________________________________________________
 ```
 
-#### 3.4.2.5. <a id="ws-end">Ending the Current Workout Session: `end`</a>
+
+#### 3.4.2.5. <a id="ws-end">Searching for a keyword in the Current Workout Session: `search`</a>
```

Comment:
Change this title to sync with the title in table of contents

Reply:
and change the anchor to <a id="ws-search">

Reply:
great catch!